nemo_aligner/models/mm/stable_diffusion/image_text_rms.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from PIL import Image
+from torchvision.transforms import CenterCrop, Compose, Normalize, Resize
+from nemo.collections.multimodal.data.clip.clip_dataset import get_preprocess_fns
+from nemo.collections.multimodal.models.vision_language_foundation.clip.megatron_clip_models import (
+    CLIPTextTransformer,
+    CLIPVisionTransformer,
+    MegatronCLIPModel,
+)
+from nemo.collections.multimodal.parts.utils import setup_trainer_and_model_for_inference
+from nemo.collections.nlp.modules.common.megatron.module import MegatronModule
+
+
+try:
+    from torchvision.transforms import InterpolationMode
+
+    BICUBIC = InterpolationMode.BICUBIC
+except ImportError:
+    BICUBIC = Image.BICUBIC
```

Comment:
why not just always use Image.BICUBIC if we're going to do this?

Reply:
good point, it is also used here https://github.com/NVIDIA/Megatron-LM/blob/a5415fcfacef2a37416259bd38b7c4b673583675/megatron/data/multimodal_dataset.py#L8

In our case, it gets it from torch transform since we are working with pytorch and its transforms module

Reply:
then can we remove the import error?